screen/home/src/main/java/com/ivy/home/HomeMoreMenu.kt

```diff
@@ -445,21 +445,25 @@ private fun QuickAccess(
                 icon = when (theme) {
                     Theme.LIGHT -> R.drawable.home_more_menu_light_mode
                     Theme.DARK -> R.drawable.home_more_menu_dark_mode
+                    Theme.TRUE_BLACK -> R.drawable.home_more_menu_dark_mode
```

Comment:
Can you add a different icon? Something from the material ones. Or the same material icon but with different fill. If doesn't look good, you can skip that

Reply:
What do you think of this way?
<img width="373" alt="Screenshot 2024-09-17 at 16 47 52" src="https://github.com/user-attachments/assets/e755cf36-724d-484e-ac8d-3911b76cb54c">

Reply:
LGTM! 👍